src/Clerk.ts

```diff
@@ -330,21 +330,16 @@ export default class Clerk {
         Logger.debug(`cookieToken: ${cookieToken}`);
         const clientUat = cookies.get('__client_uat');
         Logger.debug(`clientUat: ${clientUat}`);
-        let headerToken = (req.headers['Authorization'] ||
-          req.headers['authorization']) as string;
+        let headerToken = req.headers.authorization;
         headerToken = headerToken?.replace('Bearer ', '');
         Logger.debug(`headerToken: ${headerToken}`);
 
         // HEADER AUTHENTICATION
-        if (headerToken && !decodeToken(this, headerToken)) {
-          return signedOut();
-        }
-
-        if (!headerToken && isCrossOriginRequest(req)) {
-          return signedOut();
-        }
+        if (headerToken !== undefined) {
```

Comment:
:upside_down_face: You can probably return early here, unless I got confused.

```js
// Assuming that you want to explicitly check for undefined and not null or ''
if (headerToken === undefined) return signedOut()
if (!decodeToken) return signedOut()
// do the work to verify token and create session
```

Reply:
The purpose of this change is to take into account the `Authorization` header if exists in the incoming request.

If it doesn't (undefined) then we should move in the next steps below and probably the cookie authentication. We can't return early if the `Authorization` header doesn't exist, as the `__session` cookie might exist